packages/backend/src/db/house/houseStore.service.ts

```diff
@@ -0,0 +1,46 @@
+import { Injectable } from '@nestjs/common';
+import { InjectModel } from '@nestjs/mongoose';
+import { Model } from 'mongoose';
+import { House, HouseDocument, HouseModel } from './house.schema';
+
+@Injectable()
+export class HouseStoreService {
+  constructor(
+    @InjectModel(House.name)
+    private readonly HouseModel: Model<HouseDocument>,
+  ) {}
+
+  async create(createdHouseModel: HouseModel): Promise<House> {
```

Comment:
can you please make sure that all of these methods return HouseDocument instead of House.
Please also check userStore.service.ts
```suggestion
  async create(createdHouseModel: HouseModel): Promise<HouseDocument> {
```